documentation/asciidoc/computers/remote-access/find-your-ip-address.adoc

```diff
@@ -0,0 +1,225 @@
+[[ip-address]]
+=== How to Find your IP Address
+
+To find the local IP of your Raspberry Pi, use one of the following methods.
+
+==== Desktop
+
+Hover over the network icon in the system tray, and a tooltip will appear. This tooltip displays the name of the network you're currently connected to and your IP address.
+
+image::images/network-tooltip.png[the Network Manager tooltip displaying a WiFi network name and IP address]
+
+==== Command Line
+
+Run the following command to output your IP address to the command line:
+
+----
+hostname -I
+----
+
+==== Boot Output
+
+If you use a display with your Raspberry Pi and you boot to the command line instead of the desktop, the boot sequence will include your IP address as one of the last few output messages before your login prompt.
+
+==== Router devices list
```

Comment:
Might be worth moving this to below the raspberrypi.local section? (so that the various approaches are ordered from easiest -> most difficult)

Reply:
Agreed, moved.